src/pages/AddMembersPage/NewMemberEntry.tsx

```diff
@@ -65,7 +73,21 @@ const NewMemberEntry = ({
                   where: { address: { _ilike: addressField.value } },
                   limit: 1,
                 },
-                { name: true },
+                {
+                  name: true,
+                  ...(groupType === 'circle' && {
+                    users: [
+                      { where: { circle_id: { _eq: group.id } } },
+                      { id: true },
+                    ],
+                  }),
+                  ...(groupType === 'organization' && {
+                    org_members: [
+                      { where: { org_id: { _eq: group.organization_id } } },
```

Comment:
did you test the case where the user is a member of an org, but not the org you are adding the user to?

in this case, this doesn't work and shows an error - "existing org member" even though they don't exist.